routing/taskana-routing-rest/pom.xml

```diff
@@ -141,6 +141,18 @@
       <groupId>org.springframework</groupId>
       <artifactId>spring-test</artifactId>
     </dependency>
+    <dependency>
+      <groupId>pro.taskana</groupId>
+      <artifactId>taskana-test-api</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>pro.taskana</groupId>
+      <artifactId>taskana-common-test</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
group taskana-dependencies together.

Why do we need these dependencies?

Reply:
Same here test-api is used for the annotations, although routing-rest still depends on other common-test classes, which means we can still use the two annotations from common-test. I will remove the test-api dependency